x-pack/plugins/lists/common/schemas/common/schemas.ts

```diff
@@ -60,3 +61,37 @@ export type MetaOrUndefined = t.TypeOf<typeof metaOrUndefined>;
 
 export const esDataTypeUnion = t.union([t.type({ ip }), t.type({ keyword })]);
 export type EsDataTypeUnion = t.TypeOf<typeof esDataTypeUnion>;
+
+export const tags = DefaultStringArray;
+export type Tags = t.TypeOf<typeof tags>;
+export const tagsOrUndefined = t.union([tags, t.undefined]);
+export type TagsOrUndefined = t.TypeOf<typeof tagsOrUndefined>;
+
+export const _tags = DefaultStringArray;
+export type _Tags = t.TypeOf<typeof _tags>;
+export const _tagsOrUndefined = t.union([_tags, t.undefined]);
+export type _TagsOrUndefined = t.TypeOf<typeof _tagsOrUndefined>;
+
+// TODO: Change this into a t.keyof enumeration when we know what types of lists we going to have.
+export const exceptionListType = t.string;
+export const exceptionListTypeOrUndefined = t.union([exceptionListType, t.undefined]);
+export type ExceptionListType = t.TypeOf<typeof exceptionListType>;
+export type ExceptionListTypeOrUndefined = t.TypeOf<typeof exceptionListTypeOrUndefined>;
+
+// TODO: Change this into a t.keyof enumeration when we know what types of lists we going to have.
+export const exceptionListItemType = t.string;
```

Comment:
Are you thinking we'll default to 'simple' for now?

Reply:
This is open for debate, which is why I put the TODO. At the moment I require it, but I can default it to simple if it is not sent in if we don't know what we are using it for until we closer to the end of the UI's.